test/screenshot/spec/mdc-checkbox/fixture.scss

```diff
@@ -32,7 +32,7 @@ $custom-checkbox-color: $material-color-red-300;
 
 .custom-checkbox--container-colors {
   @include mdc-checkbox-container-colors(
-    $unmarked-stroke-color: $custom-checkbox-color,
+    // $unmarked-stroke-color: $custom-checkbox-color,
```

Comment:
was this not being used before? Should we just remove this.

Reply:
This is used in mixins/container-colors.html screenshot test file, removed the commented code.

Reply:
Cool sounds good